content/applications/productivity/iot/config/https_certificate_iot.rst

```diff
@@ -0,0 +1,269 @@
+    the "Page Not Found" code.
+
+Solution:
+    Check if `<https://www.odoo.com>`__ is down using a web browser, it's possible that it is down
+    due to maintenance.
+
+     - If `<https://www.odoo.com>`__ is down due to maintenance, unfortunately there is nothing that
+       can be done but instead wait for it to recover.
+     - If `<https://www.odoo.com>`__ is not down due to maintenance, open a
+       `support ticket <https://www.odoo.com/help>`__ on the matter. Ensure that the 3 digits status
+       code next to the code error is included in the support ticket.
+
+`ERR_IOT_HTTPS_LOAD_REQUEST_NO_RESULT`
+--------------------------------------
+
+Reason:
+    The :abbr:`IoT (Internet of Things)` box was able to reach `<https://www.odoo.com>`__ but it
+    refused to deliver the :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` certificate.
+
+Solution:
+    Ensure that the :abbr:`IoT (Internet of Things)` box and database are eligible for a
+    certificate: `Internet of Things (IoT) eligibility`_
+
+.. _iot/https-certificate:
+
+How to ensure that the HTTPS certificate is correct
+===================================================
+
+If the certificate has been applied successfully, a new :abbr:`HTTPS (Hypertext Transfer Protocol
+Secure)` :abbr:`URL (Uniform Resource Locator)` for the :abbr:`IoT (Internet of Things)` box ending
+with `.odoo-iot.com` will appear on the Odoo database.
+
+.. image:: https_certificate_iot/odoo_new_domain.png
+   :align: center
+   :alt: Odoo IoT app IoT box with .odoo-iot.com domain.
+
+When navigating to the :abbr:`URL (Uniform Resource Locator)` in a browser a secured
+:abbr:`HTTPS (Hypertext Transfer Protocol Secure)` connection will be established.
+
+.. figure:: https_certificate_iot/secured_connection.png
+   :align: center
+   :alt: Example of valid SSL certificate details on the browser.
+
+   Padlock in Chrome on Windows 10 attesting the fact that the connection is secured in :abbr:`HTTPS
+   (Hypertext Transfer Protocol Secure)`
+
+The IoT-box homepage will now display an `OK` status next to `HTTPS certificate`. The
+:guilabel:`details` button next to it will give information regarding the certificate.
+
+.. image:: https_certificate_iot/status_ok.png
+   :align: center
+   :alt: IoT box homepage with HTTPS certificate OK status.
+
+Domain Name System (DNS) issue
+==============================
+
+If the :abbr:`IoT (Internet of Things)` box can be accessed from its :abbr:`IP (Intrernet Protocol)`
+address but not the Odoo assigned domain: `.odoo-iot.com`; then the :abbr:`IoT (Internet of Things)`
+box is likely running into a :abbr:`DNS (Domain Name System)` issue. On some browsers, it will give
+an error code mentioning :abbr:`DNS (Domain Name System)` (like `DNS_PROBE_FINISHED_NXDOMAIN`).
+
+These :abbr:`DNS (Domain Name System)` issues may appear as the following in different browsers:
+
+.. tabs::
+
+   .. tab:: Chrome
+
+      .. figure:: https_certificate_iot/dns/dns_chrome.png
```

Comment:
media files for a particular .rst file should only exist in a folder with the exact same name as the .rst file - there shouldn't be sub-folders. See: https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#media-files